opencga-core/src/main/java/org/opencb/opencga/core/config/SearchConfiguration.java

```diff
@@ -157,14 +160,12 @@ public SearchConfiguration setTimeout(int timeout) {
         return this;
     }
 
-    @Deprecated
```

Comment:
Do not remove these methods. 
Leave the methods deprecated, and just call to the static method
`Configuration.reportUnusedField()`

e.g.
https://github.com/opencb/opencga/blob/e7f19667812581f14e43c403326d6c56b9c46da2/opencga-core/src/main/java/org/opencb/opencga/core/config/Configuration.java#L304-L313

Reply:
This will avoid failures in "old" configuration files that might be using the "rows" or "host" field.

Reply:
The ticket should have, in the "Communications" a message saying that this fields are removed.